src/themes/classic/components/tab.js

```diff
@@ -0,0 +1,37 @@
+const baseStyle = {
+  fontFamily: 'fontFamilies.ui'
+}
+
+const tabStyles = {
+  paddingX: '16px',
+  paddingY: '8px',
+  borderRadius: 'radii.1',
+  color: 'colors.default',
+
+  _hover: {
+    backgroundColor: 'colors.neutralAlpha.N2A',
+  },
+
+  _active: {
+    backgroundColor: 'colors.neutralAlpha.N3A'
+  },
+
+  _current: {
+    color: 'colors.blue.base',
+    backgroundColor: 'colors.blueAlpha.B3A'
+  },
+
+  _focus: {
+    boxShadow: 'shadows.focusRing'
+  }
+}
+
+const appearances = {
+  primary: tabStyles,
+  secondary: tabStyles,
```

Comment:
Does the default theme have a `secondary` appearance for tabs if they're the same?

Reply:
The default theme has two yes

Reply:
Seems like if there is no difference in appearances for v6 theme, I would move this all into `baseStyle` instead